resources/js/src/app/components/containers/LastSeenItemList.js

```diff
@@ -8,21 +10,25 @@ Vue.component("last-seen-item-list", {
             default: "#vue-last-seen-item-list"
         },
 
-        maxItems:
+        itemsPerPage:
         {
             type: Number,
-            default: App.config.itemLists.lastSeenNumber || 4
+            default: 4
         },
 
-        itemsPerPage:
+        maxItems:
         {
             type: Number,
             default: 4
         }
     },
 
     computed: Vuex.mapState({
-        items: state => state.lastSeen.lastSeenItems
+        items(state)
+        {
+            return state.lastSeen.lastSeenItems.slice(0, this.maxItems);
```

Comment:
Funktioniert das auch mit mehreren Last-Seen-Listen? `slice` entfernt die Einträge aus dem Ursprungs-Array. Könnte also passieren, dass die Artikel in der nächsten liste nicht mehr vorhanden sind.

Reply:
Im Gegensatz zu Array.splice verändert slice das Original nicht, Quelle:

https://developer.mozilla.org/de/docs/Web/JavaScript/Reference/Global_Objects/Array/slice

Test: `var arr = [1, 2, 3]; arr.slice(0, 3); console.log(arr)`